.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,9 @@
+<!--- Note: Please open the PR in draft form until you are ready for active review. -->
+Closes: <!--- Insert Issue Number(s) this PR addresses. Typing # will open a dropdown of recent issues. -->
```

Comment:
I'm not sure if the GitHub parser likes the colon here. I recently [merged a PR with `"resolves: #...."`](https://github.com/nasa/openmct/pull/4475) and it didn't automatically close the linked issue as expected, even though "resolves" is one of [GitHub's supported keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) . I had to close it manually. I've never had this issue without the colon (unless something else was wrong, like a typo in the keyword).